cranelift/codegen/src/isa/x86/unwind.rs

```diff
@@ -93,7 +95,15 @@ pub(crate) fn create_unwind_info(
                     match opcode {
                         Opcode::X86Push => {
                             let reg = func.locations[arg].unwrap_reg();
-                            unwind_codes.push(UnwindCode::SaveRegister { offset, reg });
+                            unwind_codes.push(UnwindCode::StackAlloc {
```

Comment:
Oh I see now why I'm confused as we're adding more unwind input codes.  I was expecting a `StackAlloc`  operation for the stack pointer adjustment, not for the register saves.  This is needed for the DWARF info I take it?

Reply:
DWARF and for new backend code/constructs.